### ISSUE TITLE
org:beta:create not setting default username

### ISSUE THREAD

**Issue by mshanemc** [forcedotcom/cli#1400]
### Summary

org is created, and authfile exists, but 

### Steps To Reproduce:
```
sfdx force:org:beta:create -d 1 -s -f config/project-scratch-def.json
sfdx config:list
```

### Expected result

sets defaultusername

### Actual result

didn't

### System Information

CLI Version : 
	sfdx-cli/7.136.2

 Architecture: 
	darwin-x64

 Node Version : 
	node-v16.13.2

 Plugin Version: 
	@mshanemc/sfdx-migration-automatic 0.3.1
	@oclif/plugin-autocomplete 0.3.0 (core)
	@oclif/plugin-commands 1.3.0 (core)
	@oclif/plugin-help 3.3.1 (core)
	@oclif/plugin-not-found 1.2.6 (core)
	@oclif/plugin-plugins 1.10.11 (core)
	@oclif/plugin-update 1.5.0 (core)
	@oclif/plugin-warn-if-update-available 1.7.3 (core)
	@oclif/plugin-which 1.0.4 (core)
	@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
	alias 1.2.0 (core)
	apex 0.8.0 (core)
	auth 1.8.1 (core)
	config 1.3.15 (core)
	custom-metadata 1.0.12 (core)
	data 0.6.8 (core)
	generator 1.2.1 (core)
	info 1.2.0 (core)
	limits 1.3.0 (core)
	org 1.11.1 (core)
	release-management 2.11.0
	└─ trust 1.1.0
	salesforce-alm 53.7.6 (core)
	schema 1.1.0 (core)
	sfdx-cli 7.136.2 (core)
	shane-sfdx-plugins 4.43.0
	├─ @mshanemc/plugin-streaming 1.1.7
	└─ @mshanemc/sfdx-sosl 1.1.0
	source 1.8.9 (core)
	telemetry 1.4.0 (core)
	templates 53.3.0 (core)
	user 1.7.0 (core)

 OS and Version: 
	Darwin 19.6.0

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10675890](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000mtlqnYAA/view)

**Comment by cristiand391** [comment]
This was fixed in sfdx v7.140.0
Release notes 🗒️ :  https://github.com/forcedotcom/cli/tree/main/releasenotes/sfdx#71400-march-3-2022-stable-rc
